Neos.Neos/Configuration/NodeTypes.Document.yaml

```diff
@@ -12,10 +12,15 @@
     nodeTypes:
       '*': false
       'Neos.Neos:Document': true
+  postprocessors:
```

Comment:
If we only apply the post processor to documents we need to tell integrators in the docs that they have to add it to their nodetypes. Should we apply it to all? Would that be a performance/other issue?

Reply:
It's applied to `Neos.Neos:Document` _and_ `Neos.Neos:Content`. IIRC those are the base types for everything that can be added in the Backend!?

Reply:
Ah sorry, didn't see that. Yes Document and Content should be enough